packages/component-library/src/components/layout/mt-empty-state/mt-empty-state.vue

```diff
@@ -48,24 +54,35 @@ defineEmits(["button-click"]);
 </script>
 
 <style scoped>
+.mt-empty-state {
+  display: grid;
```

Comment:
I guess flex with column direction and aligning it at the start is easier. Then you don't need to say `justify-self: start` multiple times